setup/install/storage/redis.md

```diff
@@ -6,16 +6,23 @@ sidebar:
 redirect_from: /setup/storage/redis/
 ---
 
-> :warning: We _highly_ recommend relying on [Minio](/setup/storage/minio)
-> instead of Redis if you are looking for a local persistant storage solution
-> for Spinnaker. The Redis storage implementation is untested and unsupported
-> by anyone in the Spinnaker community.
-
-> :warning: Redis can be used as Spinnaker's persistent storage source, but
-> it is unsupported and __not__ recommended for production use-cases because
-> it mixes fungible, short-lived cache entries with the Pipeline and Application
-> data that deploy all of your infrastructure. This means you will have to be
-> extra careful when clearing your Spinnaker Redis cache.
+{% include 
+   warning
+   content="We _highly_ recommend relying on [Minio](/setup/storage/minio)
+   instead of Redis if you are looking for a local persistant storage solution
+   for Spinnaker. The Redis storage implementation is untested and unsupported
+   by anyone in the Spinnaker community."
+%}
+
+{% include
+   warning
+   content="Redis can be used as Spinnaker's persistent storage source, but
+   it is unsupported and __not__ recommended for production use-cases because
```

Comment:
I would follow the format of the minio warning above and end the initial recommendation at "use-cases.", then remove the "because" and start a new sentence, "It mixes fungible, short-lived cache entries".  Just to try and avoid the slightly over-long sentence here.  Again just a suggestion and I totally defer to you and @dorbin on the final verbiage.